### PR TITLE
increased expression row height #6166

### PR DESCRIPTION
**Pull request by phoenixking25**
Addresses issue #6166

|Increased height|Original|
|----------|------|
|Increased|original|

### CODE REVIEW

File: src/components/SecondaryPanes/Expressions.css [hunk 1]
```diff
@@ -72,6 +72,7 @@
   background-color: var(--theme-body-background);
   display: block;
   position: relative;
+  min-height: 2.8em;
```
Comment (darkwing) [inline src/components/SecondaryPanes/Expressions.css]: I think we'll need to do a bit more than this; specifically, I think we should set the same height for both expression items and breakpoint items, so we'll need the height chosen to be in both places.

**Comment by phoenixking25** [comment]
|Increased height|Original|
|----------|------|
|Increased|original|

**Comment by darkwing** [comment]
Hello @phoenixking25 ; I re-read my comment and maybe wasn't specific enough.  I've added a commit along the lines of what I was thinking.

setheight

@violasong The image above sets expressions items, breakpoint items, and the checkbox rows to 2.5em height -- how does this look?

**Comment by violasong** [comment]
Thanks so much for working on this @darkwing and @phoenixking25! 

After seeing this last screenshot, I realized that while the watch expressions do match the breakpoints (yay!)  there's something off about the difference between those two rows and the two Pause on Exceptions rows, which actually look the best in their spacing. I screenshotted the PoE part and pasted it at 50% opacity over the other two parts, and it looks like both breakpoints and watch expressions have 12px more spacing between their rows. If we could tighten that up to match the PoE rows, that would be great!

**Comment by phoenixking25** [comment]
@violasong Sorry, if i am wrong but instead of tighten them up(Watch expressions and breakpoint rows). Can we increase the top margin of PoE part so that it can match the other two?

**Comment by violasong** [comment]
@phoenixking25 Sorry if this has been confusing — my comments are based on my original mockup which has the right breakpoint spacing. https://mozilla.invisionapp.com/share/TNIA9FGHER8#/screens I don't think we should change the PoE spacing as that's the one that looks perfect.  Ideally the others can match it, and they just need 12px less spacing between their rows.

**Comment by phoenixking25** [comment]
Okey, will do that thanks. :)

**Pull request merged**